### ISSUE TITLE
Motivation of cubic growth

### ISSUE THREAD

**Issue by oriolpont** [100trillionUSD/bitcoin#6]
Lack of time invariance can be explained by infectious-like spread starting at genesis time. Furthermore, cubic growth (exponent 3 power law with time) comes naturally from a susceptibility-limited infectious spread. From that, value can be modeled as standard exponent-2 Metcalfe.

Given that stock-to-flow is mostly integrative with time, the result of the model above should be roughly the same as that of the "SF model", except for the discontinuities at the "halvening" events (which do not appear well fit, nevertheless).

Thoughts?

**Comment by 100trillionUSD** [comment]
agree, s2f model is just time-model plus halving jumps. s2f and time curves are quite the same, however s2f explains better what drives valuation IMO and it allows for buy/sell timing around the halvings

**Comment by oriolpont** [comment]
As a motivator of value, s/f ratio leads to criticisms (I'm sure you've noticed them) that "the S2F model models only the supply side and ignores the demand side". 

That kind of criticism might have more or less merit, but my main concern is that it puts the focus on the argument of the power law (the s/f ratio), when IMO the key element is the use of **the power law itself**. Indeed, through Metcalfe's (classic or generalized), we are quite more describing an *adoption* proxy, which does properly refer to the *demand side*.

Another issue with putting the focus on the s/f ratio is that critics then ignore that it's an almost programmatic independent value, and consequently they treat it as a stochastic variable which must go through "cointegration tests" with the price variable. I agree with you that this kind of criticism misses the point of what S2F intends to model, but yet seasoned statisticians and modelers seemed to fall in that trap.

Anyway, should I give the viral cubic growth model a try? I should still have the equations (from four years ago lol) somewhere, and I think it would give a decent fit of the empirical curve, improving on Sornette's model, which uses a poor proxy for adoption.
